rector.php

```diff
@@ -102,18 +104,32 @@
         'rex_form_prio_element' => Form\Field\PriorityField::class,
         'rex_form_perm_select_element' => Form\Field\PermissionSelectField::class,
         'rex_form_raw_element' => Form\Field\RawField::class,
+        'rex_formatter' => Util\Formatter::class,
         'rex_i18n' => Translation\I18n::class,
+        'rex_log_entry' => Util\LogEntry::class,
+        'rex_log_file' => Util\LogFile::class,
+        'rex_logger' => Util\Logger::class,
+        'rex_markdown' => Util\Markdown::class,
+        'rex_pager' => Util\Pager::class,
+        'rex_parsedown' => Util\Parsedown::class,
         'rex_path' => Filesystem\Path::class,
         'rex_path_default_provider' => Filesystem\DefaultPathProvider::class,
+        'rex_sortable_iterator' => Util\SortableIterator::class,
         'rex_sql' => Database\Sql::class,
         'rex_sql_column' => Database\Column::class,
         'rex_sql_foreign_key' => Database\ForeignKey::class,
         'rex_sql_index' => Database\Index::class,
         'rex_sql_schema_dumper' => Database\SchemaDumper::class,
         'rex_sql_table' => Database\Table::class,
         'rex_sql_util' => Database\Util::class,
+        'rex_stream' => Util\Stream::class,
+        'rex_string' => Util\Str::class,
+        'rex_timer' => Util\Timer::class,
+        'rex_type' => Util\Type::class,
+        'rex_url' => Util\Url::class,
```

Comment:
In Skype hattest du ja vorgeschlagen, die Klasse mit in `Filesystem` zu packen, wo ich dann schrieb:

> url passt da nur so halb rein, finde ich. aber andererseits gehört die mit zu path und dem provider. daher passt schon

Bin mir da immer noch unsicher. Aber eigentlich sind `Path`, `DefaultPathProvider` und `Url` ein Trio, die zusammen gehören. Daher würde ich Url dann wohl schon mit in Filesystem packen, auch wenn es nur bedingt passt. (Ganz unpassend ist es auch nicht)

Reply:
Ist verschoben